xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,304 @@ migrate</screen>
+       </para>
+      </question>
+      <answer>
+       <para>
+	An unsupported entry in the configuration file may prevent a service
+	from starting. This, in turn, causes the node to fail at the initial
+	chef client run stage. Checking the
+	<filename>/var/log/crowbar/crowbar_join/chef.*</filename> log files on
+	the node is a good starting point.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	I need to change &ostack-bare; configuration during the upgrade but I cannot access &crow;.
+       </para>
+      </question>
+      <answer>
+       <para>
+	&crow; &wi; is accessible only when upgrade is completed or
+	when it is postponed. Postponing the upgrade can be done only after
+	upgrading all &contrnode;s using the <command>crowbarctl upgrade nodes
+	postpone</command> command. You can then access &crow; and
+	even save some modification. It is then necessary to resume the upgrade
+	using the <command>crowbarctl upgrade nodes resume</command> command
+	before you can continue with the upgrade of rest of the nodes.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Failure occurred when evacuating routers.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Check the <filename>/var/log/crowbar/node-upgrade.log</filename> file on
+	the node that performs the router evacuation (it should be mentioned in
+	the error message). The ID of the router that failed to migrate (or the
+	affected network port) is logged to
+	<filename>/var/log/crowbar/node-upgrade.log</filename>. Use the
+	&ostack-bare; CLI tools to check the state of the affected router and
+	its ports. Fix manually, if necessary. This can be done, for example,
+	by bringing the router or port up and down again. The following
+	commands can be useful for solving the issue:
+       </para>
+       <screen>
+	openstack router show <replaceable>ID</replaceable>
+	openstack port list --router <replaceable>ROUTER-ID</replaceable>
+	openstack port show <replaceable>PORT-ID</replaceable>
+	openstack port set
+       </screen>
+	<para>
+	 Resume the upgrade by running the failed upgrade step
+	again to continue with the router migration.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Upgrade status file is empty or corrupted. <command>crowbarctl upgrade status</command> reports invalid information.
+       </para>
+      </question>
+      <answer>
+       <para>
+	This can happen if the &admserv; runs out of disk space or &crow;
+	service is stopped abruptly during a status write operation. Contact
+	support for assistance on how to recover the status file.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	After performing <command>crowbarctl upgrade nodes
+	controllers</command>, some non-controller nodes weren't upgraded.
```

Comment:
Here really is "were". The point was that user might be surprised to see more nodes upgraded that was requested

Reply:
"Some nodes were not upgraded after performing <command>crowbarctl upgrade nodes controllers</command>." (keep the language simple like the above examples)

Reply:
No, this is not right, please see my original comment.